src/RadhydroSimulation.hpp

```diff
@@ -778,8 +781,9 @@ template <typename problem_t>
 void RadhydroSimulation<problem_t>::advanceHydroAtLevelWithRetries(int lev, amrex::Real time, amrex::Real dt_lev, amrex::YAFluxRegister *fr_as_crse,
 								   amrex::YAFluxRegister *fr_as_fine)
 {
+	BL_PROFILE_REGION("HydroSolver");
 	// timestep retries
-	const int max_retries = 4;
+	const int max_retries = 6;
```

Comment:
Needed (still)?

Reply:
I don't know. You'll have to test.

Reply:
Nope, 4 seems good for AMR test to pass.

Reply:
How long did it run compared to the free-fall time of the cluster?

Reply:
@BenWibking the density is 1.0. With G = 1.0, the freefall time is 0.54. The test runs for 0.5. Is it good enough?